index.html

```diff
@@ -92,7 +69,7 @@
             height="315px"
             width="430px"
              />
-            10/22/2023: 9:00 AM-12:00 PM EST
+            11/05/2023: 10:00 AM-12:00 PM EST
             </div>
             <div class="announcements-right">
               <img 
```

Comment:
Add a class name here and then apply the following css to that class name
`
.className {
  height: 100%;
  widht: 100%;
}
`